web/src/installers/index.ts

```diff
@@ -762,15 +762,15 @@ export class Installer {
     const i = new Installer();
 
     i.id = "latest";
-    i.spec.kubernetes = { version: "latest" };
-    i.spec.containerd = { version: "latest" };
-    i.spec.weave = { version: "latest" };
-    i.spec.longhorn = { version: "latest"};
-    i.spec.minio = { version: "latest"};
-    i.spec.ekco = { version: "latest" };
-    i.spec.contour = { version: "latest" };
-    i.spec.registry = { version: "latest" };
-    i.spec.prometheus = { version: "latest" };
+    i.spec.kubernetes = { version: "1.21.x" };
+    i.spec.containerd = { version: "1.4.x" };
+    i.spec.weave = { version: "2.6.x" };
+    i.spec.longhorn = { version: "1.1.x"};
+    i.spec.minio = { version: "2020-01-25T02-50-51Z"};
+    i.spec.ekco = { version: "0.12.x" };
```

Comment:
should this be latest? ekco version updates usually have important fixes for better operation of the cluster.

Reply:
@emosbaugh sure. it's weird to be the only one...  we don't support 0.x notation yet?  what options can we use to allow us to introduce a breaking change in the future?

Reply:
pinning to major notation eg 0.x would be great. additionally there have been discussions about making ekco built into kurl and no longer an addon.